irohad/synchronizer/synchronizer.hpp

```diff
@@ -39,7 +39,10 @@ namespace iroha {
 
       /**
        * Emit committed blocks
-       * Note: from one block received on consensus
+       * Note 1: from one block received on consensus
+       * Note 2: commit may be empty; in that case, it can contain empty
+       * pointer, if ledger state on this peer is up-to-date, so there is
```

Comment:
I believe "if ledger..." can be a separate sentence. Also consider replacing "so" with "then".

It is not immediately clear that there is a causation between first and second part of the statement. Maybe write something like:

"if commit is empty, it means that the ledger state is up to date..."